content/docs/contributing/release-versioning-and-terms.md

```diff
@@ -7,22 +7,74 @@ weight: 500
+
+
+| Field | Description |  CMake Global Properties |
+|---------------------|------------------------|------------------------|
+| engine_name | The name of the engine which is set to `o3de-sdk` when the SDK is created and is `o3de` in the O3DE GitHub repository source code. | | 
+| display_version | The `YY.MM.PATCH` year and month version displayed in the SDK installer and Project Manager. This value is only set when creating the SDK and is `00.00` in the O3DE GitHub repository source code.  | `O3DE_DISPLAY_VERSION_STRING` |
+| version | The `MAJOR.MINOR.PATCH` [semantic version](https://semver.org/) that is updated in the GitHub repository source code as changes are made to the APIs in the engine and can be used for general engine compatibility. Developers may use the `compatible_engines` field in their `gem.json` or `project.json` files to indicate which engine version their gem or project is known to be compatible with. | `O3DE_VERSION_STRING` `O3DE_VERSION_MAJOR` `O3DE_VERSION_MINOR` `O3DE_VERSION_PATCH`|
+| api_versions | The `MAJOR.MINOR.PATCH` [semantic version](https://semver.org/) for frameworks and tools included in the engine that are common dependencies. Developers may use the `engine_api_dependencies` field in their `gem.json` or `project.json` files to indicate which engine API versions their gem or project is known to be compatible with. | |
+| build | The incremental SDK build number that is only set when the SDK is created and is `0` in the O3DE GitHub repository source code. | `O3DE_BUILD_VERSION` |
+| O3DEVersion | Deprecated version field that has been replaced with `display_version`. |
+| O3DEBuildNumber | Deprecated build field that has been replaced with `build`. |
+
+
+Example:
+
+A developer may use the `compatible_engines` field in their `gem.json` to indicate that their gem is known to be compatible with any engine in development named `o3de` version `2.0.0` or higher or an SDK engine named `o3de-sdk` that is exactly version `1.2.0`.
+```json
+{
+    "gem_name":"CustomGem",
+    "compatible_engines": [
+        "o3de>=2.0.0",
+        "o3de-sdk==1.2.0"
+    ]
+}
+```
+
+
+### Engine APIs
+
+The API versions specified in the `api_versions` field in `engine.json` are currently based on the content inside the engine's `Code` folder and are provided as simple but useful way for developers to indicate compatibility when they expect their gem or tool to be compatibile with multiple engine versions unless breaking changes are made to APIs inside the engine's `Code` folder.
+
+| Engine API Field| Source Path |
+|---------------------|------------------------|
+| `editor` | `Code/Editor` |
+| `framework` | `Code/Framework` |
+| `launcher` | `Code/LauncherUnified` |
+| `tools` | `Code/Tools` |
+
+Example:
+
+A developer that creates a gem that includes an Editor tool, may update their `gem.json` file to indicate the gem is known to be compatibile with any non-breaking version of the Editor APIs.
```

Comment:
typo: compatibile
```suggestion
A developer that creates a gem that includes an Editor tool, may update their `gem.json` file to indicate the gem is known to be compatible with any non-breaking version of the Editor APIs.
```

Reply:
I'm very consistent at including this particular typo :smile: